src/test/resources/net/sf/jabref/importer/fileformat/MedlinePlainImporterTestDOI.bib

```diff
@@ -1,3 +1,3 @@
 @TechReport{,
-  DOI                      = {10.1016/j.cpr.2005.02.002}
+  article-id                      = {doi:10.1016/j.cpr.2005.02.002}
```

Comment:
Oh ok. The problem I thought is, that in the [medline description] (https://www.nlm.nih.gov/bsd/mms/medlineelements.html#aid)  for the AID element there come an example like this: ``AID - 10.1016/j.cpr.2005.02.002 [doi]``.
 So that ``[doi]`` has to come at the end.
That's why I thought that something like ``doi:10.1016/j.cpr.2005.02.002`` would be wrong. My question now is, whether both incidents of the doi should be possible or just one of them ?
Was this a little bit more understandable ? :)